src/App.js

```diff
@@ -13,35 +17,86 @@ Here's what a store item should look like
 
 What should a cart item look like? 🤔
 */
+//const initialStoreItems = []
 
 console.log(initialStoreItems)
 
 export default function App() {
+  const [storeItems, setstoreItems] = useState(initialStoreItems)
+  const [cartItems, setCartItems] = useState([])
   // Setup state here...
+  // add item to cart
+  //create new array that inludes the items
+   //checks if item already exists in cart
+   // if exists returns true,
+    //define quantity
+  const addItemToCart = item => {
+    console.log("push items", item)
+    const exists = cartItems.find(cartItem => cartItem.id === item.id)
+    if (exists) {
+      const updateCartItem = cartItems.map(selectedItem => {
```

Comment:
Nice way to check if the item exists in the cart

Reply:
I also like how you have named things so we know exactly what each part is for/doing